src/CMakeModules/Bootstrap_Linux.cmake

```diff
@@ -59,38 +58,19 @@ FIND_PACKAGE (X11 REQUIRED)
 
 if (ENABLE_HTML)
     if (USE_SYSTEM_CEF)
-        set(CEF_LIB_PATH "/usr/lib/casparcg-cef-117")
-        set(CEF_INCLUDE_PATH "/usr/include/casparcg-cef-117")
-
-        set(CEF_LIB
-            "-Wl,-rpath,${CEF_LIB_PATH} ${CEF_LIB_PATH}/libcef.so"
-            "${CEF_LIB_PATH}/libcef_dll_wrapper.a"
-        )
+        find_package(CEF 117 REQUIRED)
```

Comment:
with this being find_package(CEF 117 REQUIRED), what would happen if I had libcef117 and libcef138 installed on my machine? would it be able to find the correct one?

Reply:
It will pick up 117. `find_package` looks at the config file exported by libcef (via my patches) and the config file says:
```cmake
write_basic_package_version_file(${CMAKE_CURRENT_BINARY_DIR}/CEFConfigVersion.cmake
  VERSION ${PROJECT_VERSION_MAJOR}
  COMPATIBILITY SameMajorVersion
)
```
So, it will reject 138.
If you want, I can spin up Debian package for libcef138 and upload it to my PPA. You can install both and try it. They can be installed side by side too.

Reply:
no thats fine, Ill take your word for it :)

Reply:
@Julusian as a side note, I don't see version 138 😃 Did you mean 128 maybe?...